src/data/internals-telemetry-dashboard.yaml

```diff
@@ -0,0 +1,40 @@
+name: Telemetry Dashboards
+summary: Help us with public data at https://telemetry.mozilla.org and some of its hosted dashboards.
+introduction: |
+  ## About the Telemetry Portal
+  
+  The [Telemetry Portal](https://telemetry.mozilla.org/) is a one-stop shop for indexing public and mozilla-specific
+  data dashboards and documentation. There you can explore aggregated Firefox anonymous usage statistics on the
+  Measurement Dashboard, learn what data is collected on which versions in the Probe Dictionary, and more.
+  
+  ## Who Works On the Telemetry Portal?
+  
+  The Firefox Telemetry team, a group of developers spanning timezones UTC+2 to UTC-4, maintain and develop the index,
+  documentation, and some of the tools and dashboards hosted on https://telemetry.mozilla.org. Most of the time we work
```

Comment:
Not sure how this is rendered in the end and if it auto-detects links. If not, this should be <https://telemetry.mozilla.org>